support/Pipelines/Bbh/Inspiral.py

```diff
@@ -55,6 +56,77 @@ def inspiral_parameters(
     }
 
 
+def _load_spec_id_params(id_params_file: Path) -> dict:
+    """Load SpEC initial data parameters from 'ID_Params.perl'."""
+    # Here we have to deal with SpEC storing the initial data parameters in a
+    # perl script (yay!). We evaluate the perl script, emit the variables as
+    # YAML, and load that into Python as a dictionary.
+    perl_variables = [
+        "$ID_rExcA",
+        "$ID_rExcB",
+        "\@ID_cA",
+        "\@ID_cB",
+        "$ID_Omega0",
+        "$ID_adot0",
+        "$ID_MA",
+        "$ID_MB",
+        "\@ID_chiA",
+        "\@ID_chiB",
+        "$ID_chiAMagnitude",
+        "$ID_chiBMagnitude",
+    ]
+    perl_script = [
+        f'do "{id_params_file.resolve()}";',
+        "print YAML::Dump({ "
```

Comment:
Is this part of the core perl install? I don't know how easy it is to install perl modules locally. It may be easier to parse the file ourselves rather than rely on a perl module. We then also don't have to worry about calling into perl from a python script :)

Reply:
I don't think so? I just tried on my local machine a `test.pl` that had only `use YAML` and it complained about a missing module. [This](https://metacpan.org/pod/YAML) also does warn against using the `YAML` module if we were going to use a module.

Reply:
Ok I wanted to avoid parsing perl syntax with regex because I hate that too :D I'll see what can be done

Reply:
Could you just read the file in as a string, do some replacements like you're already doing and use a yaml parser in python?

Reply:
Convert perl syntax to yaml or json? Yea good idea

Reply:
Basically, yeah

Reply:
Just rewrite all of SpEC to use yaml. Easy ;) :D

Reply:
Ok done (@knelli2's suggestion, not @nilsdeppe's 😄)

Reply:
> Just rewrite all of SpEC to use yaml. Easy ;) :D

Actually that's kind of what we're doing ;)